Utilities/DataLoading/EdFi.LoadTools/SmokeTest/SdkTests/DestructiveMethodsGenerator.cs

```diff
@@ -29,7 +29,9 @@ public class DestructiveMethodsGenerator : ITestGenerator
             _dependenciesRetriever = dependenciesRetriever;
             _dependenciesSorter = dependenciesSorter;
         }
-        
+
+        protected ILog Log => LogManager.GetLogger(GetType().Name);
```

Comment:
I was originally going to ask why you went with declaring the logger as `protected` since we typically have it `private readonly`, but then I found a few instances of it being protected, so this can stay.